zeebe/exporters/camunda-exporter/pom.xml

```diff
@@ -190,6 +190,12 @@
       <scope>test</scope>
     </dependency>
 
+    <dependency>
+      <groupId>io.camunda</groupId>
+      <artifactId>zeebe-protocol-impl</artifactId>
+      <scope>test</scope>
+    </dependency>
```

Comment:
💭 Personally, I'd leave this out, as it's only use is to access few strings that should probably be part of the zeebe-protocol itself. Those can simply be re-defined in the test as constants.

Reply:
Thanks for the feedback, Nico! I've opted to replace the references to `UserTaskRecord` constants with string literals, as we currently have only a few tests where these attribute names are used. 

Initially, I used the constants from `UserTaskRecord` because `zeebe-protocol-impl` was available as a transitive dependency via `zeebe-protocol-test-util`. However, `maven-dependency-plugin` failed the build requiring to define it explicitly, so I've removed the definition of `zeebe-protocol-impl` dependency from the `camunda-exporter` test scope.